features/lorawan/lorastack/phy/loraphy_target.h

```diff
@@ -18,35 +18,46 @@
 #ifndef LORAPHY_TARGET
 #define LORAPHY_TARGET
 
+#define EU868 0
+#define AS923 1
+#define AU915 2
+#define CN470 3
+#define CN779 4
+#define EU433 5
+#define IN865 6
+#define KR920 7
+#define US915 8
+#define US915_HYBRID 9
+
 #ifdef MBED_CONF_LORA_PHY
```

Comment:
Are you expecting here that an unknown identifier gets resolved to EU868 ? Because I think ,
```
MBED_CONF_LORA_PHY == ANYTHING
```
would resolve to zero.

Reply:
Not the way the C processor works, sadly. It first does macro substitution, and inside a `#if`, any unknown identifiers are converted to `0`. So with `MBED_CONF_LORA_PHY` defined as `SILLY` it starts with

    #if MBED_CONF_LORA_PHY == EU868
    #elif MBED_CONF_LORA_PHY == KR920
    #else 
    #endif

then converts this to

    #if 0 == 0
    #elif 0 == 1
    #else 
    #endif

There may be some more macro magic with concatenation you could do to specially distinguish literal `0`.